test-apps/display-test-app/src/frontend/DisplayTestApp.ts

```diff
@@ -16,38 +16,23 @@ import { showStatus } from "./Utils";
 import { Dock } from "./Window";
 
 const configuration: DtaConfiguration = {};
-
-// Retrieves the configuration for starting SVT from configuration.json file located in the built public folder
-async function retrieveConfiguration(): Promise<void> {
-  return new Promise<void>((resolve, _reject) => {
-    if (ProcessDetector.isMobileAppFrontend) {
-      if (window) {
-        const urlParams = new URLSearchParams(window.location.hash);
-        urlParams.forEach((val, key) => {
-          (configuration as any)[key] = val;
-          Object.assign(configuration, { iModelName: urlParams.get("iModelName") });
-        });
-      }
-      const newConfigurationInfo = JSON.parse(window.localStorage.getItem("imodeljs:env")!);
-      Object.assign(configuration, newConfigurationInfo);
-      resolve();
-    } else {
-      const request: XMLHttpRequest = new XMLHttpRequest();
-      request.open("GET", "configuration.json");
-      request.setRequestHeader("Cache-Control", "no-cache");
-      request.onreadystatechange = ((_event: Event) => {
-        if (request.readyState === XMLHttpRequest.DONE) {
-          if (request.status === 200) {
-            const newConfigurationInfo: any = JSON.parse(request.responseText);
-            Object.assign(configuration, newConfigurationInfo);
-            resolve();
-          }
-        }
+const getFrontendConfig = () => {
+  // Why???
+  if (ProcessDetector.isMobileAppFrontend) {

```

Comment:
@markschlosseratbentley any idea on the why?

Reply:
@khanaffan Is this code path needed still?

Reply:
@markschlosseratbentley did you receive an answer from @khanaffan?

Reply:
I shouldn't have broken anything here was moreso wondering if it was used. If no objections, I'll leave it as is so we can merge this PR.

Reply:
Somebody needs to test it on iOS.

Reply:
Are there any instructions, I don't see any in the readme? I'm on my Mac today and have an iPad I could use...

Reply:
I always have to ask @khanaffan or @markschlosseratbentley for instructions for building and running on iOS. Better if display team tests anyway (but readily available instructions e.g. in the README would be great).

Reply:
Figured it out, testing now. Just need to run `build:ios` and then open `test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app.xcodeproj/project.pbxproj` in Xcode and run it.

Reply:
**@pmconne: "@markschlosseratbentley did you receive an answer from @khanaffan?"**
No. @khanaffan Is this code path still needed?

Reply:
@calebmshafer When I try to build this for iOS (`npm run build:ios`) , I get the following error. Did you successfully build this branch?
```
ERROR in /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/node-fetch@2.6.4/node_modules/node-fetch/lib/index.mjs 1157:15-18
Can't import the named export 'URL' from non EcmaScript module (only default export is available)
 @ /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/@azure+core-http@2.2.0/node_modules/@azure/core-http/dist-esm/src/nodeFetchHttpClient.js
 @ /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/@azure+core-http@2.2.0/node_modules/@azure/core-http/dist-esm/src/defaultHttpClient.js
 @ /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/@azure+core-http@2.2.0/node_modules/@azure/core-http/dist-esm/src/coreHttp.js
 @ /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/@azure+storage-blob@12.8.0/node_modules/@azure/storage-blob/dist-esm/storage-blob/src/index.js
 @ /Users/markschlosser/imodeljs/core/backend/lib/CloudStorageBackend.js
 @ /Users/markschlosser/imodeljs/core/backend/lib/imodeljs-backend.js
 @ ./lib/backend/Backend.js
 @ ./lib/backend/MobileMain.js
npm ERR! code ELIFECYCLE

```

Reply:
Hmm I did not hit that error. Let me check again.

Reply:
I just did a full clean & rebuild and didn't hit any build errors.

Reply:
Thanks Caleb... I'll try to do a full clean.

Reply:
Ok.  I did a reclone and it builds now.  However, I am running into trouble running the app (opening an iModel).  Did you successfully open an iModel, @calebmshafer ?
Once it asks me to login to my Bentley ID, I get:

```
libc++abi: terminating with uncaught exception of type NSException
dyld4 config: DYLD_LIBRARY_PATH=/usr/lib/system/introspection DYLD_INSERT_LIBRARIES=/Developer/usr/lib/libBacktraceRecording.dylib:/Developer/usr/lib/libMainThreadChecker.dylib:/Developer/Library/PrivateFrameworks/DTDDISupport.framework/libViewDebuggerSupport.dylib
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Completion handler passed to -[imodeljs_test_app.ViewController webView:runJavaScriptAlertPanelWithMessage:initiatedByFrame:completionHandler:] was not called'
terminating with uncaught exception of type NSException
```

Reply:
FYI, I get the same error trying to authenticate this app using a build based on master.

Reply:
Thanks @markschlosseratbentley. Mind if we merge this then and we can circle back to fix mobile DTA once #2152 has been merged? We should be able to get that merged tomorrow fairly early in the day.

Reply:
@pmconne going to merge this if no objections.

Reply:
@calebmshafer Sounds fine. We will have to figure out what is going on, on both master and here, though.